Makefile

```diff
@@ -155,7 +157,7 @@ copy_node_js:
 ifeq ($(IN_DOCKER),)
 	$(warning Command is designed to be run in the container)
 endif
-	for dest in $(NODE_LIBS_JS) ; do cp $(NODE_MODULES)$$dest $(STATIC_JS) ; done
+	for dest in $(NODE_LIBS_JS) ; do cp -r $(NODE_MODULES)$$dest $(STATIC_JS) ; done
```

Comment:
> Would it be possible to make this explict rather than copying entire directories? I'd have concerns that this way we don't really know what we're putting on the CDN.

It's possible, it'd just mean more that needs to be updated in the makefile each time.  I'll make the change.

> I suppose an alternative would be to use the lists in settings to do the copying but that's a bit of a yak-shaving excercise.

That would be a little more difficult because the lists reference the destination of the files (`node_lib`) rather than the source.

I thought about trying to reference them directly from the node folders in the lists (a relative reference to files), rather than copying at all, but a) seemed fragile on different platforms where the `node_modules` folder might be somewhere different and b) I wasn't sure if having the source in one place for reference was desirable rather than having to go wading through `node_modules`.